include/openenclave/bits/defs.h

```diff
@@ -149,4 +149,16 @@
 #define OE_DEPRECATED(FUNC, MSG) FUNC
 #endif
 
+/* OP-TEE provides single-threaded enclaves only, and its ELF loader does not
+ * support thread-local relocations. Hence, any enclave that includes a
+ * variable marked with __thread will not only not work, but it will fail to
+ * load altogether.
+ */
+#if defined(_ARM_) || defined(_M_ARM) || defined(__arm__) || \
+    defined(__thumb__) || defined(__aarch64__)
+#define OE_THREAD_LOCAL_STORAGE
```

Comment:
I think my previous comment got lost.
Can we instead just define __thread to empty string for ARM? That would avoid editing 3rd party code.

Reply:
Hadn't thought of that! I'll try it.

Reply:
Removed `OE_THREAD_LOCAL_STORAGE` and did as you suggested. Thanks for the idea!